CONTRIBUTING.md

```diff
@@ -48,21 +57,14 @@ pytest --cov-report html:htmlcov
 
 ## Code Style Guide
 
-Use [google standards](https://google.github.io/styleguide/pyguide.html)
-
-## Linter
-
-The code style/formatting will be checked with
-[`black`](https://black.readthedocs.io/en/stable/) and
-[`flake8`](https://flake8.pycqa.org/en/latest/) in the CI
+Use [google standards](https://google.github.io/styleguide/pyguide.html).
 
 ## Checklist for code contributions
 
 - [ ] Branch off of `main`
 - [ ] Develop the feature or fix
 - [ ] Write tests to cover all use cases
+- [ ] Ensure all checks pass (`pre-commit`)
 - [ ] Ensure all tests pass (`pytest`)
 - [ ] Ensure test coverage is >95%
-- [ ] Autoformat (`black .`)
-- [ ] Spellcheck your code and docstrings
```

Comment:
I figured we should practice what we preach, and in the process why not automate it.